PIMbench/cpp-gemm/gemm.cpp

```diff
@@ -207,7 +207,7 @@ void gemm(uint64_t row, uint64_t colA, uint64_t colB, std::vector<std::vector<in
 int main(int argc, char *argv[])
 {
   struct Params params = getInputParams(argc, argv);
-  std::cout << "Row: " << params.row << " Column: " << params.columnA << "\n";
+  std::cout << "Running gemm on PIM for matrix row: " << params.row << " column: " << params.columnA << "\n";
```

Comment:
I think we can also print params.columnB here since columnA and columnB are different by default.